assignment-sweet-compose/answer/docker-compose.yml

```diff
@@ -6,13 +6,12 @@
 version: '2.4'
 
 services:
-
   traefik:
     image: traefik:1.7-alpine
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock
     ports:
-      - "8080:80"
+      - "80:80"
```

Comment:
I didn't get how it works, we open the port `80`, but `- traefik.port=8080`. For me it works, but it's a little bit confusing

Reply:
Like I mentioned in Q&A, this publishes the proxy listening port on host port 8080. That's not related to the Dashboard inside the proxy container listening on 8080, which we "proxy the dashboard" with Traefik itself like: `localhost:8080->traefik:80->traefik:8080`